### PR TITLE
Don't fail if NETWORK_POLICY_PROVIDER is unset

### PR DESCRIPTION
**Pull request by caseydavenport**
Fixes issue from this comment: https://github.com/kubernetes/kubernetes/pull/26017/files/35289c764954011e69d289a20ccb26fd3b0f1062#r68873333

@ixdy Haven't had a chance to test this against AWS, but I think it fixes your issue.

### CODE REVIEW

File: cluster/common.sh [hunk 1]
```diff
@@ -476,13 +476,13 @@ function yaml-quote {
...
-  if [[ -n "${NETWORK_POLICY_PROVIDER}" ]]; then
+  if [[ -n "${NETWORK_POLICY_PROVIDER:-}" ]]; then
     appends="extensions/v1beta1=true,extensions/v1beta1/networkpolicies=true"
   fi
 
   # Generate the RUNTIME_CONFIG.
-  if [[ -n ${appends} ]]; then
-    if [[ -n ${RUNTIME_CONFIG} ]]; then
+  if [[ -n ${appends:-} ]]; then
+    if [[ -n ${RUNTIME_CONFIG:-} ]]; then
```
Comment (thockin) [inline cluster/common.sh]: pedanticly, this should be quote-wrapped.

**Comment by caseydavenport** [comment]
@thockin - As per the conversation here: https://github.com/kubernetes/kubernetes/issues/28401#issuecomment-230306775 should we just always enable this and get rid of all the if/then logic?

**Comment by thockin** [comment]
yes, I think so.

**Comment by caseydavenport** [comment]
OK

@thockin - I ended up removing all of this in https://github.com/kubernetes/kubernetes/pull/28549, so closing in favor of that.

**Pull request closed without merging**